sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/AvroGenericRecordToStorageApiProtoTest.java

```diff
@@ -266,6 +268,28 @@ enum TestEnum {
           .noDefault()
           .endRecord();
 
+  private static final Schema SCHEMA_WITH_MAP;
+
+  static {
+    ImmutableMap<String, Object> defaultVal =
+        ImmutableMap.<String, Object>builder().put("key1", "value1").build();
+
+    SCHEMA_WITH_MAP =
+        SchemaBuilder.record("TestMap")
+            .fields()
+            .name("nested")
+            .type()
+            .optional()
+            .type(BASE_SCHEMA)
+            .name("aMap")
+            .type()
+            .map()
+            .values()
+            .stringType()
+            .mapDefault(defaultVal)
+            .endRecord();
+  }
+
```

Comment:
```suggestion
  private static final Schema SCHEMA_WITH_MAP =
      SchemaBuilder.record("TestMap")
          .fields()
          .name("nested")
          .type()
          .optional()
          .type(BASE_SCHEMA)
          .name("aMap")
          .type()
          .map()
          .values()
          .stringType()
          .mapDefault(ImmutableMap.<String, Object>builder().put("key1", "value1").build())
          .endRecord();
```

Can we do this all inline instead?

Reply:
sure